pkg/kubernetes/helm.go

```diff
@@ -315,6 +308,11 @@ func (h *HelmExecute) runHelmPackage() error {
 	if h.verbose {
 		helmParams = append(helmParams, "--debug")
 	}
+	if len(h.config.AppTemplates) > 0 {
```

Comment:
Is there a particular reason to carry over the kubectl deployment method from the `kubernetesDeploy` into the `helmExecute`?

Reply:
Here we use ```AppTemplates``` as a slice of strings where each string is a name of values yaml file. 
Values file contains, for example,  ```image``` field with template value (```image: {{ .Values.image.repository }}```), so that values from ```config.yml``` would be inserted in the values file, after that helm command would be executed.